src/components/Contentful/DynamicHeroBackground.vue

```diff
@@ -0,0 +1,116 @@
+<template>
```

Comment:
Please rename so this is generic and doesn't allude to some dependency on the Hero component.